app.py

```diff
@@ -20,6 +21,8 @@
     transform_loans_data,
     transform_main_chart_data,
 )
+from typing import Dict, Set
```

Comment:
Please, run `isort` command https://pycqa.github.io/isort/ for this file
p.s. this package is already install, just run command to format `imports`